cbdr/cortex
# COR-762: Fix Chrome Tag Box

## Purpose:
Removes the weird box line around the Tag Field Box in Chrome. This was the result of a chore specific style for a box shadow around the bootstrap tags input field. It is gone now.

> There's a weird glitch in Chrome where there's a faint box still around the tag field. Can we try to remove?

## JIRA:
https://cb-content-enablement.atlassian.net/browse/COR-762

## Steps to Take On Prod
* Go to New Employer Blog or Media in Chrome
* See there is no line around the tags box

## Changes:
* Changes to setup
  * N/A

* Architectural changes
  * N/A

* Migrations
  * N/A

* Library changes
  * N/A

* Side effects
  * N/A

## Screenshots
* Before
<img width="592" alt="screen shot 2017-04-24 at 11 39 45 am" src="https://cloud.githubusercontent.com/assets/8419757/25348169/fa1999c2-28e2-11e7-8e30-9aed9731226b.png">

* After
<img width="595" alt="screen shot 2017-04-24 at 11 38 53 am" src="https://cloud.githubusercontent.com/assets/8419757/25348171/fcda7bf4-28e2-11e7-9790-cc74f18cafb2.png">

## QA Links:
http://web.cortex-6.development.c66.me/

## How to Verify These Changes
* Specific pages to visit
  * New Employer Blog or Media

* Steps to take
  * See there is no line around the box

* Responsive considerations
  * N/A

## Relevant PRs/Dependencies:
N/A

## Additional Information
N/A
